monitoring/opencensus/requirements.txt

```diff
@@ -0,0 +1,45 @@
+appdirs==1.4.4
```

Comment:
Are all of these requirements needed by the sample? Please delete any that the sample does not explicitly need to function.

Reply:
Pruned, thank you.